docs/user_guide.rst

```diff
@@ -259,6 +259,49 @@ optimizations. Let us try it out on our decomposed and routed circuit above:
     gates for a particular device. In the example above we don't have them, however it is generally a good idea to run
     decomposition once again after the simplification.
 
+Classical control
+-----------------
+Some IQM quantum computers support classically-controlled gates, that is, gates that are executed
+conditionally depending on the result of a measurement preceding them in the quantum circuit. This
+support currently has several limitations:
+
+* Only the ``x``, ``y``, ``rx``, ``ry`` and ``r`` gates can be classically controlled.
+* The gates can only be conditioned on one classical bit, and the only control available is to
+  apply the gate if the bit is 1, and apply an identity gate if the bit is 0.
+* The availability of the controlled gates depends on the instrumentation of the quantum computer.
+
+The classical control can be applied on a circuit operation using :meth:`~cirq.Operation.with_classical_controls`:
+
+.. code-block:: python
+
+    import cirq
+    q1, q2 = cirq.NamedQubit('Alice'), cirq.NamedQubit('Bob')
+    circuit = cirq.Circuit(cirq.X(q1), cirq.measure(q1, key = 'Alices_Measurement_Register'),
+              cirq.X(q2).with_classical_controls('Alices_Measurement_Register'), cirq.measure(q2))
+    circuit
```

Comment:
```suggestion
    q1, q2 = cirq.NamedQubit('Alice'), cirq.NamedQubit('Bob')
    circuit = cirq.Circuit(
        cirq.X(q1),
        cirq.measure(q1, key = 'A'),
        cirq.X(q2).with_classical_controls('A'),
        cirq.measure(q2),
    )
    print(circuit)
```
The measurement keys should not be called registers IMO, because the actual qubits in them can overlap. This is a more general system than in Qiskit.